Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build the manager binary
-FROM golang:1.18 as builder
+FROM golang:1.19 as builder
```

Comment:
Let's base our builder images on ubi8

```
FROM registry.access.redhat.com/ubi8/go-toolset:1.18 as builder
```

Reply:
Hmm, the updated SDK (the OpenShift version) requires Go 1.19, which doesn't seem available with UBI yet.